src/main/webapp/data/userMapData.json

```diff
@@ -162,6 +163,7 @@
       "Cegep de Bois-de-Boulogne",
       "Chetwynd Secondary School",
       "Collège Ahuntsic",
+      "College de Bois-de-Boulogne",
```

Comment:
This looks similar to `Cegep de Bois-de-Boulogne` 3 lines above; might it be the same institution?